proto/cosmos/crypto/secp256k1/keys.proto

```diff
@@ -29,10 +49,18 @@ message PubKey {
   bytes key = 1;
 }
 
-// PrivKey defines a secp256k1 private key.
-message PrivKey {
+// PrivKeyOld defines a tendermint secp256k1 private key.
+message PrivKeyOld {
   option (amino.name)             = "tendermint/PrivKeySecp256k1";
   option (amino.message_encoding) = "key_field";
 
   bytes key = 1;
 }
+
+// PrivKey defines a comet uncompressed secp256k1 private key.
```

Comment:
I don't think we have (un)compressed private keys 😄

Reply:
Yeah but porting the changes from here https://github.com/0xPolygon/cometbft/blob/develop/crypto/secp256k1/secp256k1.go#L22 where we explicitly call `PrivKeyName    = "comet/PrivKeySecp256k1Uncompressed"`
What do you suggest as a comment?

Reply:
WDYS @sergio-mena ?

Reply:
Sorry for delay coming here. In the PR I'm working on to upstream these changes to (vanilla) CometBFT I'm not using "uncompressed" because I felt similar to what motivated this thread: I renamed it to `cometbft/PrivKeySecp256k1Eth`.
Let me know if you'd like me to open two PRs (one here and one in peppermint) to keep the naming consistent.

Reply:
Yes, please.
Thanks @sergio-mena